Makefile

```diff
@@ -170,6 +170,9 @@ deploy: .version kustomize ## Deploy controller to the K8s cluster specified in
 undeploy: ## Undeploy controller from the K8s cluster specified in ~/.kube/config.
 	$(KUSTOMIZE) build config/${OVERLAY} | kubectl delete --ignore-not-found -f -
 
+# Let .version be phony so that a git update to the workarea can be reflected
```

Comment:
You and your 80-character limitations. Turn that off

Reply:
79 characters is ideal